template/api/product/models/product.settings.json

```diff
@@ -35,11 +35,6 @@
     "productDescription": {
```

Comment:
I know it's not related to this PR, but I think it would be cleaner to rename this one to just "description". We already know that we're talking about the product, and it would be more aligned with other properties. For example, it's "title" and not "productTitle"

Reply:
agreed 👍